src/service-object.ts

```diff
@@ -92,6 +92,12 @@ export interface ServiceObjectConfig {
    * object is Bucket.
    */
   parent: ServiceObjectParent;
+
+  /**
+   * For long running operations, how often should the client poll
+   * for completion.
+   */
+  pollInterval?: number;
```

Comment:
Only idea would be to change to pollIntervalMs to be clear of the expected value.